plonetheme/onegov/resources/sass/components/controlpanel/controlpanels.plone.scss

```diff
@@ -110,3 +110,21 @@ body[class*=-controlpanel] #columns .standalone {
     background-color: #ccc;
   }
 }
+
+// This page duplicates #container in the DOM when a different portlet manager
+// is selected (a plone issue). So we need to hide visual clutter with CSS.
+body.template-topbar-manage-portlets #container #container {
```

Comment:
Is it correct that there is twice the id `container`? Is that the repetition of it?

Reply:
Exactly! It catches when the DOM repeats itself (in a nest fashion)

Reply:
Thanks!